docs/go/testing.md

```diff
@@ -245,3 +247,17 @@ func (s *UnitTestSuite) Test_ProgressWorkflow() {
 	s.Equal(value, 100)
 }
 ```
+
+## Debugging
+
+You can use a debugger tool provided by your favorite IDE to debug your Workflow Definitions prior to testing or executing them.
+
+Note that when debugging your Workflow code, the Temporal Go SDK includes deadlock detection which fails a Workflow Task in case the code blocks over a second without relinquishing execution control.
```

Comment:
OK.
```suggestion
The Temporal Go SDK includes deadlock detection which fails a Workflow Task in case the code blocks over a second without relinquishing execution control.
```